init.lua

```diff
@@ -271,7 +276,7 @@ function stamina.set_sprinting(player, sprinting)
 	if settings.sprint_particles and sprinting then
 		local pos = player:getpos()
 		local node = minetest.get_node({x = pos.x, y = pos.y - 1, z = pos.z})
-		if node.drawtype ~= "airlike" then
+		if node.drawtype ~= "airlike" and node.drawtype ~= "liquid" and node.drawtype ~= "flowingliquid" then
```

Comment:
FYI: `node.drawtype` is always `nil`, thus it will always pass the checks. Look up `node.name` in `minetest.registered_nodes` which contains the drawtype information.

Reply:
Thanks for catching that, should be fixed now.